src/field/mobile-app-ui/index.md

```diff
@@ -0,0 +1,157 @@
+
+- **Source**: internal GPS of the mobile device or [external](../external_gps/) GPS receiver connected via Bluetooth
+- **Longitude, Latitude**: current position
+- **X, Y**: current position in project's coordinate reference system
+- **Horizontal** and **Vertical accuracy** of the GPS position
+- **Altitude**: ellipsoidal height if internal GPS is used. [External GPS](../external_gps/) devices usually return orthometric heights (ellipsoid with the geoid separation applied).
+- **Satellites (in use/view)**: number of satellites
+- **Speed**
+- **Last fix**: time of the last received GPS position
+- **GPS antenna height** that can be set in [GPS settings](#gps-settings)
+
+
+### Sync
+The **Sync** button can be used to synchronise changes during the field survey.
+
+If automatic synchronisation of changes is allowed in the [Settings](#streaming-mode-and-recording-settings), this button will indicate that synchronisation is in progress.
+
+![Mergin Maps mobile app Sync button](./mobile-app-sync.jpg "Mergin Maps mobile app Sync button")
+
+
+### Add
+Tap the **Add** button to enter the *recording mode* so you are able to [survey new features](../mobile-features/). 
+![Mergin Maps mobile app Add button](./mobile-app-add-button-record-mode.jpg "Mergin Maps mobile app Add button") 
+
+The **active layer** currently used for the survey is displayed on the top of the screen. Tap on it to choose another active layer from editable layers in the project.
+![Active survey layer](./mobile-app-active-layer.jpg "Active survey layer")
+
+:::tip
+[How to Add, Edit, Delete Features](../mobile-features) will show you how to capture points, lines and polygons in <MobileAppName /> as well as how to edit their geometry and attributes.
+:::
+
+### Layers
+Tap the **Layers** button to display the layers in your project. 
+
+Here, you can toggle specific layers on and off to change their visibility on the map, see layers legend, and browse features in the layers. 
+
+More details can be found in [Layers in Mergin Maps Mobile App](../layers/).
+
+![Mergin Maps mobile app Layers](./mobile-app-layers.jpg "Mergin Maps mobile app Layers")
+
+
+### More options: Zoom to project, Map themes, Position tracking, Local changes, Settings
+The **More** button opens a list of additional options
+
+![Mergin Maps mobile app More button](./mobile-app-more-options.jpg "Mergin Maps mobile app More button")
+
+- **Projects**: a short cut to the main page of the <MobileAppNameShort />
+- [**Zoom to project**](../../gis/features/#project-extent): zoom to the extent of all visible layers within the project. The project extent can be set in [QGIS in Project Properties](../../gis/features/#project-extent).
+- [**Map themes**](../../gis/setup_themes/): a list of map themes set up in a <MainPlatformName /> project in QGIS
+- [**Position tracking**](../tracking/): to start tracking of your tracks during the field survey
+- **Local changes**: the overview of changes yet to be synchronised
```

Comment:
Let's make sure people understand those are my local changes (not those on a server)

```suggestion
- **Local changes**: the overview of your local changes to be synchronised
```